docs/openapi-fetch/api.md

```diff
@@ -143,6 +143,18 @@ const { data, error } = await client.PUT("/submit", {
 });
 ```
 
+::: tip
+
+For convenience, `openapi-fetch` sets `Content-Type` to `application/json` automatically
+for any request that provides value for the `body` parameter. When the `bodySerializer` returns an instance of `FormData`,
+`Content-Type` is omitted, allowing the browser to set it automatically with the correct message part boundary.
+
+You can also set `Content-Type` manually through `headers` object either in the fetch options,
+or when instantiating the client. Setting `Content-Type` to `null` will omit the header, however the native fetch API
+will likely set it to `text/plain` in this case.
```

Comment:
Not sure we really need to mention the `null` thing anymore, when this behaves properly now. 🤷‍♂️ 

Opinions?

Reply:
Yeah just remove it 🔪

Reply:
BTW Just a thought... I wonder whether allowing the `bodySerializer` to return `{contentType: "...", body: ...}` wouldn't be the best way to allow transport customization 🤔